.github/workflows/snapcraft.yml

```diff
@@ -0,0 +1,25 @@
+name: 🧪 Test snap can be built on x86_64
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: snapcore/action-build@v1
+        id: build
+
+      - uses: diddlesnaps/snapcraft-review-action@v1
+        with:
+          snap: ${{ steps.build.outputs.snap }}
+          isClassic: 'false'
+          # Plugs and Slots declarations to override default denial (requires store assertion to publish)
+          # plugs: ./plug-declaration.json
+          # slots: ./slot-declaration.json
```

Comment:
That workflow, is just to test if things are fine or not. So would you want me to do? Can you clarify? Actually, all your messages a more descriptive, rather than saying what should I/You do. Thanks

Reply:
That's my bad, I was thinking we remove the action you added and just use the build one in the file I had linked. Thinking about it again though it probably would be wise to run through the `review-tools` Snap on every MR - could we just avoid using that action though?

I'd prefer to just create a job that runs `snap install review-tools` and then runs `review-tools` though. On that note though is the `review-tools` Snap documented anywhere? I saw that it was published by Canonical, but I haven't seen anywhere that they recommend running it. If it is recommended somewhere I'd be more than fine including it though.

All the rest of your changes looked fine, it was just this one part that I wanted looked at.

Reply:
The snap review aren't that much, but do one thing, test the snap with this commit, in a real machine, I think you're right. There is an issue with self-contained apps and libadwaita.